engine/network/client/IgeNetIoClient.ts

```diff
@@ -28,9 +28,9 @@ import { IgeEventingClass } from "../../core/IgeEventingClass";
 // };
 
 export interface IgeNetIoClientOptions {
-	connectionRetry: boolean;
-	connectionRetryMax: number;
-	reconnect: boolean;
+	connectionRetry?: boolean;
+	connectionRetryMax?: number;
+	reconnect?: boolean;
```

Comment:
Maybe we should be setting some sensible defaults here... I wasn't sure... what do you think?

Reply:
I've not really used the networking stuff so far so I'm not sure, but there's definitely a cleanup task needed later to go through and try to eliminate as much usage of ? as possible, and also to tackle the use of `any`